nemo/collections/nlp/utils/evaluation_utils.py

```diff
@@ -0,0 +1,151 @@
+# =============================================================================
```

Comment:
How about change the name to analysis_utils.py? what do you think? or eval_analysis_utils.py

Reply:
I like the name "evaluation_utils"